docker-compose.yml

```
@@ -7,86 +7,83 @@ services:
-    networks: [fusionnet]
+    networks:
+      - fusionnet
 
-  # Graph database for the knowledge graph.  Neo4j is used by default; the
-  # NEO4J_AUTH environment variable should be updated for production.
   neo4j:
     image: neo4j:5.15
     environment:
-      - NEO4J_AUTH=${NEO4J_USER:-neo4j}/${NEO4J_PASSWORD:-change_this_password}
+  NEO4J_AUTH: "${NEO4J_USER:?}/${NEO4J_PASSWORD:?}"
     ports:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)